### ISSUE TITLE
Add message to describe why a word is black-listed

### ISSUE THREAD

**Issue by DerZyklop** [kyaido/eslint-plugin-literal-blacklist#9]
We love this package in our project! Thank you! 

It would be awesome to have the possibility to describe why a term is black-listed. So that the devs can see it when they hover over the black-listed word.

### Possible implementation 

In `.eslintrc.json`, there is a `string[]` inside of `literal-blacklist/literal-blacklist`. Change it to array of `{term: string, message: string}` instead of `string`.

```
"literal-blacklist/literal-blacklist": [
	"error", [
		{
			"term": "Teilnehmer",
			"message": "we use »Teilnehmende« to be gender-neutral",
		},
		{
			"term": "ebay",
			"message": "The brand is »eBay«, not »ebay« ;)",
		},
	],
],
```

Plugin code would change to something like this:

```
      options.forEach(option => {
        if(value.indexOf(option.term) !== -1) {
          message = `You should not use "${option.term}".\n${option.message}`;
          context.report({node: node, message: message});
        }
      });
```

PS:
Maybe there is a way to make it backwards compatible to still allow `string[]` AND `{term: string, message: string}[]` instead of only `{term: string, message: string}[]`.

**Comment by DerZyklop** [comment]
This should work and should be backwards compatible:

```javascript
    options.forEach(option => {
      if(value.indexOf(option) === -1) return;
    
      const optionIsString = typeof option === 'string' || option instanceof String;
      const term = optionIsString ? option : option.term;
    
      message = `You should not use "${term}".`;
      if(!optionIsString) message = `message\n${option.message}`;
    
      context.report({node: node, message: message});
    });
```

**Comment by DerZyklop** [comment]
Maybe @kyaido could explain the reason for `value.indexOf(option) === -1` to me. It was in the original code. I did not understand it, hence don’t know if I implemented it correctly or if option.term also needs to be checked against -1.

**Comment by kyaido** [comment]
@DerZyklop 
Thank you for great suggestion. It sounds good!
I will fix in a few days.

**Comment by kyaido** [comment]
@DerZyklop 
I released v1.0.0.
You can use it with object option 😃
